bpf/bpf_lxc.c

```diff
@@ -433,8 +435,7 @@ static __always_inline int handle_ipv6(struct __ctx_buff *ctx, __u32 *dstID)
 	return ipv6_l3_from_lxc(ctx, &tuple, ETH_HLEN, ip6, dstID);
 }
 
-declare_tailcall_if(__or(__and(is_defined(ENABLE_IPV4), is_defined(ENABLE_IPV6)),
-			 is_defined(DEBUG)), CILIUM_CALL_IPV6_FROM_LXC)
+declare_tailcall_if(ENABLE_LXC_TAILCALLS, CILIUM_CALL_IPV6_FROM_LXC)
```

Comment:
I would probably have kept original conditions and just change to `__or(ENABLE_LXC_TAILCALLS, $PREV_COND))`.

Reply:
I agree this would be nice documentation, but if the condition is going to be hardcoded to 1 then it doesn't make such a big difference. We can always look back into git or do a revert if we want to switch back to the old behaviour. This is also limited to v1.8 branch so the latest will continue to document the real intent here.